Content.Server/ParticleAccelerator/EntitySystems/ParticleAcceleratorSystem.ControlBox.cs

```diff
@@ -396,4 +411,16 @@ private void OnUIRescanMessage(EntityUid uid, ParticleAcceleratorControlBoxCompo
 
         UpdateUI(uid, comp);
     }
+    private bool CheckAccess(EntityUid uid, EntityUid user, ParticleAcceleratorControlBoxComponent comp)
```

Comment:
use `Entity<ParticleAcceleratorControlBoxComponent> ent, EntityUid user` so its harder to mix up argument entities
then use ent instead of uid and ent.Comp instead of comp